test_data/mysql_text_report.txt

```diff
@@ -1,7 +1,7 @@
 ----------------------------
 Summary of Conversion
 ----------------------------
-Schema conversion: POOR (65% of 17006 columns mapped cleanly) + some missing primary keys.
+Schema conversion: POOR (29% of 17006 columns mapped cleanly) + some missing primary keys.
```

Comment:
Same comment. What changed in the complexity calculation logic?

Reply:
Previously missing primary key was not included in the calculation of total number of warnings but now they are included and this impacted the rating of the schema